checkov/cdk/checks/python/ECRImmutableTags.yaml

```
@@ -0,0 +1,13 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_51
+  name: Ensure ECR Image Tags are immutable
+  category: GENERAL_SECURITY
+scope:
+  languages:
+    - python
+definition:
+  pattern: aws_cdk.aws_ecr.Repository(<ANY>)
+  conditions:
+    - not_pattern: aws_cdk.aws_ecr.Repository(<ANY>, image_tag_mutability=aws_cdk.aws_ecr.TagMutability.IMMUTABLE, <ANY>)
```

WDYT about this one? 
```suggestion
  pattern: aws_cdk.aws_ecr.Repository(<ANY>, image_tag_mutability=$ARG, <ANY>)
  conditions:
    - metavariable: $ARG
    - not_pattern: aws_cdk.aws_ecr.TagMutability.IMMUTABLE

```